src/screens/app/Components/CommentsTouchBox/CommentsTouchBox.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
-import {TouchableWithoutFeedback} from 'react-native-gesture-handler';
 import {Caption} from 'react-native-paper';
-import {StyleSheet} from 'react-native';
+import {StyleSheet, TouchableOpacity} from 'react-native';
```

Comment:
I was getting a Direction Error from TouchableWithoutFeedback so I removed it. If we don't want feedback for this we can just style the opacity to 1.

Reply:
Hmm, thats weird. The error should be fixed since I fixed it last time but its okay